### PR TITLE
[#7684] Relocate and change menu for "+" icon on home/chat screen

### LINKED ISSUE #7684: Relocate and change menu for "+" icon on home/chat screen
### User Story

As a user, I want my options to be more obvious to me on the home/chat screen so that I can start chats more easily.

### Description

*Type*: Feature

*Summary*: In #7534, we're separating DApps into their own tab and renaming the `Home` tab to `Chat`. 

The new `Chat` screen will be redesigned to have the `+` icon in the bottom center (rather than top right), with fewer options.

#### Expected behavior
`+` button from current Home screen is relocated to bottom center. Bottom sheet is used to give users the additional chat options, `Start new chat`, `Start group chat`, etc.

There are no longer options relating to DApps. 

[DESIGNS](https://www.figma.com/file/deXIBZm6fU1CCL0voYbfTujz/Navigation?node-id=1%3A2)

#### Actual behavior
The current Home screen forces users to search for browser items from hard-to-notice `+` icon in upper right corner.

### PR DESCRIPTION
**Pull request by flexsurfer**
https://youtu.be/txLKVCgInEs

fixes #7684

### CODE REVIEW

File: src/status_im/accounts/create/core.cljs [hunk 1]
```diff
@@ -107,13 +107,14 @@
 
 (defn account-set-name [{{:accounts/keys [create] :as db} :db now :now :as cofx}]
   (fx/merge cofx
-            {:db                                  (assoc db :accounts/create {:show-welcome? true})
+            {:db                                              db
              :notifications/request-notifications-permissions nil
-             :dispatch                            [:navigate-to :home]}
+             :dispatch-n                                      [[:navigate-to :home]
```
Comment (jeluard) [inline src/status_im/accounts/create/core.cljs:112]: 2 navigate?

File: src/status_im/ui/screens/home/filter/views.cljs [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+            [status-im.i18n :as i18n]
+            [status-im.ui.components.colors :as colors]
+            [status-im.ui.screens.home.views.inner-item :as inner-item]
+            [status-im.utils.utils :as utils]
+            [status-im.ui.components.animation :as animation]
+            [reagent.core :as reagent]
+            [re-frame.core :as re-frame]
+            [status-im.ui.components.icons.vector-icons :as icons]))
+
+(defn search-input [_ {:keys [on-cancel on-focus on-change]}]
```
Comment (jeluard) [inline src/status_im/ui/screens/home/filter/views.cljs:14]: Are those 2 files new code?

Comment (flexsurfer) [reply src/status_im/ui/screens/home/filter/views.cljs:14]: no just moved filter in the new namespace

File: src/status_im/accounts/create/core.cljs (hunk 1 above)
Comment (flexsurfer) [reply src/status_im/accounts/create/core.cljs:112]: yes, we need to open home and immediately open modal welcome screen

**Review by jeluard — APPROVED** [review]

**Review by dmitryn — APPROVED** [review]

**Comment by asemiankevich** [comment]
@Serhy  @antdanchenko please rework tests here based on changes:

after entering name (new account):
- new screen Get started shown with new button 
- Chats screen has Invite friends button and wording. Plus button is relocated

**Comment by Serhy** [comment]
Adjusted autotests to expect new welcome screen
**Fat finger issue:**
I've tested on my Android 8 and iOS, the new screen and want to note one thing: I often misstapped the chat action-button from bottom sheet. That's because I tap the very end of text of the item. 
And indeed actionable area is this:
Screenshot 2019-03-19 at 10 17 04

Id say it makes sense to increase an actionable area of the element at the bottom 'New Chat' sheet  at least +50px more or more. (Develop build has the actionable area on 'New Chat' screen stretched till the end of screen)

**Comment by flexsurfer** [comment]
good catch @Serhy , thanks

**Comment by errorists** [comment]
Yes, the entire width of every list item there should be tappable, thank you so much for spotting this @Serhy 🙏

**Comment by flexsurfer** [comment]
fixed

**Comment by errorists** [comment]
Speaking of which, it would be *really* sweet if we could adjust the touchable behaviour on list items to something like this instead

ScreenFlow

**Comment by flexsurfer** [comment]
@Serhy could you take a look, 1 test failed

File: src/status_im/ui/screens/home/styles.cljs [hunk 3]
```diff
@@ -168,55 +169,51 @@
...
    :align-items      :center
-   :justify-content  :center})
-
-(def spinner-container
-  {:margin-right 10})
+   :justify-content  :center
+   :shadow-offset    {:width 0 :height 2}
+   :shadow-radius    8
+   :shadow-opacity   1
+   :shadow-color     "rgba(0, 12, 63, 0.12)"
```
Comment (errorists) [inline src/status_im/ui/screens/home/styles.cljs]: @flexsurfer can we please change those to 
:shadow-offset    {:width 0 :height **1**}
:shadow-radius    **6**
:shadow-opacity   1
:shadow-color     "rgba(0, 12, 63, **0.2**)"

this is to closer match the appearance with the shadow of the tab bar, thx! everything else is good! 🙏

**Review by errorists — APPROVED** [review]

**Comment by asemiankevich** [comment]
@flexsurfer faucet is down it is known and not related to this PR

**Comment by flexsurfer** [comment]
both comments from Maciej have been implemented

**Pull request merged**